e2e-tests/README.md

```diff
@@ -33,3 +33,12 @@ contracts/adder$ NODE_URL=wss://ws.dev.azero.dev AUTHORITY="$THE_SEED" ./deploy.
 e2e-tests$ RUST_BACKTRACE=1 SUDO_SEED="$THE_SEED" NODE_URL=wss://ws.dev.azero.dev:443 \
   ADDER=$DEPLOY_ADDRESS ADDER_METADATA=../contracts/adder/target/ink/metadata.json cargo test adder -- --nocapture
 ```
+
+## Running on feature net
+
+Run a feature net by adding an appropriate label to a pull request, ie `trigger:create-featurenet`
+or `trigger:create-hotfix-featurenet`, then after its started run
+
+```bash
+e2e-tests$ NODE_URL=wss://ws-fe-a0-1564.dev.azero.dev:443 cargo test finalization
```

Comment:
Question: is the address always the same? If not, how do I get it after running the featurenet? Maybe just link to the relevant documentation.

Reply:
good point, added a note about that